rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section22/rule_22_22.json

```diff
@@ -9,10 +9,10 @@
             "rule_id": "22-22",
             "ruleset_reference": "Table G3.5.3",
             "rule_description": "The baseline chiller efficiencies shall be modeled at the minimum efficiency levels for full load, in accordance with Tables G3.5.3.  ",
-            "applicable_rmr": "Baseline Model",
+            "applicable_rmr": "Baseline RMR",
             "rule_assertion": "=",
             "comparison_value": "Expected Value",
-            "primary_rule": "Full",
+            "primary_rule": "Yes",
```

Comment:
Are all these rule test uses Full instead of Yes?

Reply:
@jugonzal07 Could you verify if this change is appropriate? Have we changed the ```primary_rule``` key from ```Full``` to ```Yes```?